.travis.yml

```diff
@@ -1,19 +1,17 @@
 language: go
 go:
-  - "1.11"
+  - "1.13"
 env:
-  - DEP_VERSION="0.5.4" CLOUDSDK_CORE_DISABLE_PROMPTS=1
+  - CLOUDSDK_CORE_DISABLE_PROMPTS=1
 install:
-  - curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-257.0.0-linux-x86_64.tar.gz | tar xfz - -C $HOME
-  - $HOME/google-cloud-sdk/bin/gcloud components install app-engine-python app-engine-go cloud-datastore-emulator
-  # TODO: lock a version of mockgen.
-  - go get github.com/golang/mock/mockgen
-  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
-  - chmod +x $GOPATH/bin/dep
+  - curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-307.0.0-linux-x86_64.tar.gz | tar xfz - -C $HOME
+  - $HOME/google-cloud-sdk/bin/gcloud components install beta cloud-datastore-emulator
+  # JRE is needed for the datastore emulator
+  - sudo apt-get install openjdk-8-jre-headless
```

Comment:
Emulator is not starting in travis build with "Unable to prepare the data directory for the emualtor" error. I think you saw https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/144 and  decided to install JRE 8 , right?


Add   - java -version    after the installation, maybe it will be pointing to 1.7 even after install, and you need to do something like sudo update alternatives  to explicitly specify 1.8

Reply:
No, this was a side effect of another issue, which is now fixed (process was hanging around stuck).